src/pages/en/guides/baas/appwriteio.mdx

```diff
@@ -0,0 +1,777 @@
+---
+title: Appwrite & Astro
+description: Add authentication, storage, and realtime data to your Astro application
+layout: ~/layouts/BackendAsAServiceLayout.astro
+service: Appwrite
+stub: false
+---
+import FileTree from '~/components/FileTree.astro'
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
+
+
+
+[Appwrite](https://appwrite.io/) is a self-hosted backend-as-a-service platform that provides developers with all the core APIs required to build any application.
+
+## Integrating with Astro
+
+In this section, we'll use the [Appwrite JavaScript SDK](https://appwrite.io/docs/getting-started-for-web#getSDK) to connect your Appwrite project to Astro.
+
+### Prerequisites
+
+To get started, you will need to have the following:
+
+1. **An Astro project** - If you don't have an Astro project yet, our [Installation guide](/en/install/auto/) will get you up and running in no time.
+
+2. **An Appwrite instance, or hosted instance**. If you don't have an account, you can [sign up](https://cloud.appwrite.io/login) for a free account and create a new Appwrite project. You can also use an existing project if you have one. 
+
+3. **Appwrite credentials** - You can find the following credentials in your Appwrite console **Project Overview > API key**. 
+
+    - **Project ID** - The ID of your Project (not the name). 
+
+### Setting up credentials
+
+To add your Appwrite project's credentials to Astro, create an `.env` file in the root of your project with the following variables:
+
+:::tip
+You may find that you will need additional environment variables, feel free to continue adding to the `.env` file.
+:::
+
+```ini title=".env"
+PUBLIC_PUBLIC_APPWRITE_ENDPOINT=YOUR_ENDPOINT
+PUBLIC_PUBLIC_APPWRITE_PROJECT_ID=YOUR_PROJECT_ID
+PUBLIC_PUBLIC_APPWRITE_DB_ID=YOUR_DATABASE_ID
+PUBLIC_APPWRITE_BLOG_ID=YOUR_BLOG_COLLECTION_ID
+PUBLIC_APPWRITE_COMMENTS_ID=YOUR_COMMENTS_COLLECTION_ID
+PUBLIC_APPWRITE_BUCKET_ID=YOUR_STORAGE BUCKET_ID
```

Comment:
Where do we find these? I don't think we can find them until we create a blog collection